### PR TITLE
Recycler code cleanup: move js code into admin.js file

### PR DESCRIPTION
**Pull request by dgvirtual**

### CODE REVIEW

File: themes/Admin/js/admin.js [hunk 1]
```diff
@@ -22,3 +22,21 @@ if (selectAllElement) {
         toggleSelectAll(e.target);
     });
 }
+
+
+// function for recycler get requests issued directly from select box
+function sendGetRequest(selectedValue) {
```
Comment (lonnieezell) [inline themes/Admin/js/admin.js]: The name of this function sounds like a generic function to send a get request, but it's tied specifically to the Recycler. Please rename to something more specific.

Comment (dgvirtual) [reply themes/Admin/js/admin.js]: Good point, I will change it to something more specific.

**Comment by dgvirtual** [comment]
Function name changed

**Comment by lonnieezell** [comment]
That works. Thanks!

**Pull request merged**